components/openweather_api/openweather_api.app.mjs

```diff
@@ -1,11 +1,97 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "openweather_api",
-  propDefinitions: {},
+  propDefinitions: {
+    appId: {
+      type: "string",
+      label: "App Id",
+      description: "Please refer to [Doc here](https://home.openweathermap.org/api_keys) for your app ID.",
+    },
```

Comment:
Since we can get the api key from `this.$auth.api_key`, we don't need the `appId` prop.
```suggestion
```